baselines/aad.md

```diff
@@ -188,52 +180,60 @@ The methods **AAD Certificate-Based Authentication (CBA)**, **FIDO2 Security Key
 
 The option **Federal PIV card (federated from agency on-premises Active Directory Federation Services or other identity provider)**, although technically phishing-resistant presents significant risks if the on-premises authentication infrastructure (e.g. ADFS) is compromised. Therefore federated PIV is not a preferred option and agencies should migrate to the options listed in the preferred section above. If an agency does use an on-premises PIV authentication and federate to AAD, reference the [guidance at this link](https://playbooks.idmanagement.gov/piv/network/group/) to enforce PIV logon via AD group policy.
 
+<!--Policy: MS.AAD.3.1v1; Criticality: SHALL -->
 - _Rationale:_ The security risk of allowing weaker forms of MFA is that they do not protect against sophisticated phishing attacks. By enforcing methods which are resistant to phishing those risks are minimized.
 - _Last modified:_ June 2023
 
 #### MS.AAD.3.2v1
 If phishing-resistant MFA has not been enforced yet, then an alternative MFA method SHALL be enforced for all users.
 
+<!--Policy: MS.AAD.3.2v1; Criticality: SHALL -->
 - _Rationale:_ This is a backup security policy to help protect the tenant in the event that phishing-resistant MFA has not been enforced yet. This policy requires that MFA is enforced and thus reduces the risks of single form authentication.
 - _Last modified:_ June 2023
 - _Note:_ If a conditional access policy has been created that enforces phishing-resistant MFA, then this policy is not necessary. This policy does not dictate the specific MFA method.
 
 #### MS.AAD.3.3v1
 If phishing-resistant MFA has not been enforced yet and Microsoft Authenticator is enabled, it SHALL be configured to show login context information.
 
+<!--Policy: MS.AAD.3.3v1; Criticality: SHALL -->
```

Comment:
Rego has Should/Not-Implemented.  I see control reads as SHALL so I assume it should be SHALL/NOT-IMPLEMENTED.  Need issue for Rego?